app/src/org/commcare/models/database/InterruptedFormState.java

```diff
@@ -59,6 +63,10 @@ public FormIndex getFormIndex() {
         return formIndex;
     }
 
+    public boolean getInterruptedDueToSessionExpiration(){
```

Comment:
nit: rename to `isInterrupted..`

Reply:
[dec7568](https://github.com/dimagi/commcare-android/pull/2858/commits/dec75686a49af9085f4b2889769c0a1c812ae85a)